defaults/main.yml

```diff
@@ -8,7 +8,9 @@ haproxy_group: haproxy
 haproxy_frontend_name: 'hafrontend'
 haproxy_frontend_bind_address: '*'
 haproxy_frontend_port: 80
+haproxy_frontend_ssl_port: 443
 haproxy_frontend_mode: 'http'
+haproxy_ssl_pem: false
```

Comment:
For the default, please change this to an empty string.

Also, can you add a line in the README.md file documenting this variable so first-time users of the role know to add a path to their pem file on the local/Ansible workstation?